sway/sway-ipc.7.scd

```diff
@@ -1371,6 +1371,9 @@ available:
 |- 0x80000014
 :  bar_status_update
 :  Send when the visibility of a bar should change due to a modifier
+|- 0x80000015
```

Comment:
Since i3 won't implement such an event, it may be wise to leave 0x80000015 for a future i3 event, and choose a much higher event code.

Reply:
`bar_status_update` is also Sway specific. I just added this one right after. Should they both be changed?